app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -529,6 +530,10 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
             )
         }
 
+        // Set webView background color to transparent to override default black and default white backgrounds
+        // and let css draw its own background when its ready
+        webView.setBackgroundColor(ResourcesCompat.getColor(resources, R.color.colorLaunchScreenBackground, theme))
```

Comment:
This is the only new change in this pr, that is not part of home-assistant/android/pull/1938